### PR TITLE
[stable/nginx-ingress]Better default comment

### PR DESCRIPTION
**Pull request by naseemkullah**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
This may seem silly, but after going from DS to deployment. This is a more preferable out of the box setting as it allows you to have more replicas than there are nodes if required. Essentially it is a softer requirement then the previous one.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/nginx-ingress/values.yaml [hunk 1]
```diff
@@ -124,14 +124,16 @@ controller:
...
     # podAntiAffinity:
-    #   requiredDuringSchedulingIgnoredDuringExecution:
-    #   - labelSelector:
-    #       matchExpressions:
-    #       - key: app
-    #         operator: In
-    #         values:
-    #         - nginx-ingress
-    #     topologyKey: "kubernetes.io/hostname"
+    #   preferredDuringSchedulingIgnoredDuringExecution:
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/values.yaml:131]: how about we include both? this way a user can decide which they prefer while also providing a quick way of setting them. I would also add a blurb on top of affinity that points to the kubernetes docs that helps users understand these options.

Comment (naseemkullah) [reply stable/nginx-ingress/values.yaml:131]: Sure, done.

Comment (ChiefAlexander) [reply stable/nginx-ingress/values.yaml:131]: This is super nit-picky so apologies... Could you format the reference to the docs like we do for the other vars
https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L136-L138

Comment (naseemkullah) [reply stable/nginx-ingress/values.yaml:131]: You are right to ask for that. Done!

**Pull request merged**